Add GitHub actions

jp-dbt opened the pull request:
This PR adds github actions for dbt-server

lostmygithubaccount commented on .github/actions/notify-dbt-runtime/action.yml:
```diff
@@ -0,0 +1,63 @@
+name: "Notify dbt Runtime"
```
how does this notify us?

racheldaniel reviewed (approved):
Great! Comparing to `internal-analytics`, we've added two secrets to replace some hard-coded values, and removed the staging endpoints, but that's it-- correct?

jp-dbt commented:
> Great! Comparing to `internal-analytics`, we've added two secrets to replace some hard-coded values, and removed the staging endpoints, but that's it-- correct?

Correct!
